operations/patch_util.go

```diff
@@ -241,6 +248,23 @@ func (p *patchParams) validatePatchCommand(ctx context.Context, conf *ClientSett
 		}
 	}
 
+	// Validate trigger aliases exist
+	if len(p.TriggerAliases) > 0 {
+		validTriggerAliases, err := comm.ListPatchTriggerAliases(ctx, p.Project)
+		if err != nil {
+			return nil, errors.Wrap(err, "error fetching trigger aliases")
+		}
+		invalidAliases := []string{}
+		for _, alias := range p.TriggerAliases {
+			if !utility.StringSliceContains(validTriggerAliases, alias) {
+				invalidAliases = append(invalidAliases, alias)
+			}
+		}
+		if len(invalidAliases) > 0 {
+			return nil, errors.Errorf("trigger aliases %v are not defined for project '%s'", invalidAliases, p.Project)
```

Comment:
would it help if we instead displayed the valid aliases for the project?

Reply:
done.
Though the commit isn't showing in the PR yet 🤔 . GitHub status shows issues with PRs a the moment.